MultipleFileUpload/Model/MySQL/Queue.php

```diff
@@ -0,0 +1,162 @@
+<?php
```

Comment:
Nette\Database implementation was not in last release (1.1.0) donwloaded by composer thus I did not notice it is in master branch by Zdeněk Jurka. Probably it solve most of issues with nette 2.4? (Nette\Environment, etc.)